R/todo_comment_linter.R

```diff
@@ -42,23 +42,18 @@
 #' @export
 todo_comment_linter <- function(todo = c("todo", "fixme")) {
   todo_comment_regex <- rex(one_or_more("#"), any_spaces, or(todo))
```

Comment:
While at this: Do we want to lint rpxygen TODO comments as well?

`#' TODO` wouldn't lint ATM.

Reply:
makes sense, though it's a bit odd for people to be leaving these comments in their documentation. 

OTOH, it's common: https://github.com/search?q=lang%3AR%20%2F%23%27%5Cs*TODO%2F&type=code

Let's push to follow-up though as the real fix (#2439) is based here and I'd rather not deal with more merge conflicts 😂

Reply:
#2447